readme.md

```diff
@@ -230,6 +229,23 @@ You can execute the plugin to validate the api files without breaking the build
 | jsonOutputFile | File | ${rest-guide-validator.outputDir}/validationReport.json | Output file for JSON validation report.  | rest-guide-validator.jsonOutputFile | 
 | groupBy   | rule / file | rule | Specify how you want to group the violation output                                                                          | rest-guide-validator.groupBy |
 
+# Standalone belgif-rest-guide-validator
```

Comment:
this section is buried within the Maven documentation. If you start reading the README, you'd still think it's a plugin that requires Maven.
Best to open README with scope of the tool (without naming it plugin), mentioning choice Maven plugin or standalone executable;  then separate chapter per choice.

Reply:
Done